src/lib/viewers/media/DashViewer.js

```diff
@@ -12,6 +12,8 @@ const CSS_CLASS_HD = 'bp-media-controls-is-hd';
 const SEGMENT_SIZE = 5;
 const MAX_BUFFER = SEGMENT_SIZE * 12; // 60 sec
 const MANIFEST = 'manifest.mpd';
+const DEFAULT_VIDEO_WIDTH = 854;
```

Comment:
I like appending the unit, e.g. `DEFAULT_VIDEO_WIDTH_PX`

Reply:
Done